.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  dispatch-update:
+    name: Dispatch Update
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send request
+        if: ${{ github.repository_owner == 'Submitty' }}
+        run: >
+          curl -L
+          -X POST
+          -H "Accept: application/vnd.github+json"
+          -H "Authorization: Bearer ${{ secrets.PAT }}"
+          -H "X-GitHub-Api-Version: 2022-11-28"
+          https://api.github.com/repos/${{ github.repository_owner }}/Submitty/dispatches
+          -d '{
+            "event_type": "repo-release",
+            "client_payload": {
+              "repo_name": ${{ toJSON(github.event.repository.name) }},
+              "repo": ${{ toJSON(github.event.repository.full_name) }},
+              "tag": ${{ toJSON(github.event.release.tag_name) }}
+            }
+          }'
```

Comment:
Is there a way to use the GH API for this instead of manually making a request with curl?